plugins/CancelDrag/CancelDrag.js

```diff
@@ -7,7 +7,7 @@ export function CancelDragPlugin() {
 		}
 		drop({ cancel, dispatchSortableEvent, originalEvent, dragEl, cloneEl }) {
 			// In case the 'ESC' key was hit, the originalEvent is of type 'dragEnd' (except for macOS chrome, where that will be sent if the element is dropped while pressing a key, like CMD, so we just disable this cancelling/reverting if swapping).
-			if (originalEvent?.type === 'dragend' && !this.options.swap) {
+			if (originalEvent?.type === 'dragend' && !this.options.swap && !this.options.sort) {
```

Comment:
The issue is macOS Chrome will send the `dragend` event, and I was hoping that since the integration sortables have `sort:false` but the kanban doesn't, it would work, but it disables the canceling everywhere, unfortunately.